calabash/Classes/FranklyServer/Routes/LPIntrospectionRoute.m

```diff
@@ -0,0 +1,56 @@
+#if ! __has_feature(objc_arc)
+#warning This file must be compiled with ARC. Use -fobjc-arc flag (or convert project to ARC).
+#endif
+//
+//  LPAccessorRoute.m
+//  calabash
+//
+//  Created by Chris Fuentes on 6/5/15.
+//  Copyright (c) 2015 Xamarin. All rights reserved.
+//
+
+#import "LPIntrospectionRoute.h"
+#import "UIScriptParser.h"
+#import "LPTouchUtils.h"
+#import "LPJSONUtils+Introspection.h"
+
+@interface LPIntrospectionRoute ()
+@end
+
+@implementation LPIntrospectionRoute
```

Comment:
Is this missing the:

```
- (BOOL) supportsMethod:(NSString *) method atPath:(NSString *) path;
```

method?

Prompted by your questions about how routing is done, I dug into this a bit.  It seems to me that if this method is not being called by the router, it should be.